app/views/layouts/_subnav.html.erb

```diff
@@ -12,7 +12,7 @@
         <%= active_link_to "Episodes", podcast_episodes_path(@podcast), active: /\/episodes/, class: "nav-link" %>
         <%= active_link_to "Feeds", podcast_feeds_path(@podcast), active: /\/feeds/, class: "nav-link" %>
         <% if Rails.env.development? %>
-          <%= active_link_to "Metrics", podcast_metrics_path(@podcast), active: /\/metrics/, class: "nav-link" %>
+          <%= active_link_to "Metrics", podcast_metrics_path(@podcast), active: /\/podcasts\/[0-9]+\/metrics/, class: "nav-link" %>
```

Comment:
now that `episodes/guid-1234/metrics` is a route, needed to specify this regex more to keep it from highlighting.